### DISCUSSION TITLE
[Feature] Open link in mobile app

### DISCUSSION THREAD

**Discussion by FreeWind6** [immich-app/immich#13907] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If you send a link to a photo or album, it opens through the browser. It would be cool if the link was opened through the Immich app

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2327
